gamemode/modules/animations/sh_animations.lua

```diff
@@ -185,6 +185,7 @@ local function AnimationMenu()
     AnimFrame.btnMinim:SetVisible(false)
     AnimFrame:SetVisible(true)
     AnimFrame:MakePopup()
+    AnimFrame:ParentToHUD()
```

Comment:
This previously parented to `Panel` above. Are you trying to remove the need for the panel?

Reply:
I had some weird shit happen with ParentToHUD when not called behind MakePopup.

Reply:
> I had some weird shit happen

This is more superstition than a reason. Why don't you run `ParentToHUD` on the panel?